src/analyzers/jetanalyzer.jl

```diff
@@ -1,7 +1,34 @@
 """
-    struct JETAnalyzer <: AbstractAnalyzer
-
-JET.jl's default error analyzer.
+Every [entry point of error analysis](@ref jetanalysis-entry) can accept
+any of [general JET configurations](@ref JET-configurations) as well as
+the following additional configurations that are specific to the error analysis.
+
+---
+- `mode::Symbol = :basic`:\\
+  Switches the error analysis pass. Each analysis pass reports errors according to their
+  own "error" definition.
+  JET by default offers the following modes:
+  - `mode = :basic`: the default error analysis pass.
+    This analysis pass is tuned to be useful for general Julia development by reporting common
+    problems, but also note that it is not enough strict to guarantee that your program never
+    throws runtime errors.\\
+    See [`BasicPass`](@ref) for more details.
+  - `mode = :sound`: the sound error analysis pass.
+    If this pass doesn't report any errors, then your program is assured to run without
+    any runtime errors (unless JET's error definition is not accurate and/or there is an
+    implementation flaw).\\
```

Comment:
This seems like too strong of a guarantee: errors triggered for particular values (e.g., #311), or an error hidden behind runtime dispatch, presumably won't be caught.